sale_packaging_price/models/sale_stock.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
Quoting from there:

> File naming: [...]
> models/< inherited_main_model >.py

And class name is correct in CamelCase, also as guidelines say.